contracts/Multi-signature/ApprovalCollector.sol

```diff
@@ -0,0 +1,215 @@
+pragma solidity ^0.4.24;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "./MultiplexingContract.sol";
+
+
+/**
+ * @title DelegateERC20 smart contract
+ * @dev Controls execution of ERC20 token transactions
+ */
+contract ApprovalCollector is Ownable {
+    using SafeMath for uint256;
+
+    MultiplexingContract private multiplexingContract;             //address of controlled contract
+    uint256 constant CALL_LIFETIME = 300000;    //transaction lifetime in seconds
```

Comment:
```suggestion
    uint256 constant TRANSACTION_LIFETIME = 300000; // in seconds
```
For the value of the constant, please confirm with Vlad.

Reply:
Done